funnel/views/index.py

```diff
@@ -34,6 +34,20 @@ def funnelindex():
     return {'projects': projects}
 
 
+@app.route('/account')
+@funnelapp.route('/account')
+@lastuser.requires_login
+def account():
+    return render_template('account.html.jinja2')
+
+
+@app.route('/contact_exchange')
+@funnelapp.route('/contact_exchange')
```

Comment:
Whatever name we use here has to be in the reserved names list.

Reply:
It also needs to be a carefully considered name that accounts for future features. I'd say `contacts` is more appropriate.

Reply:
Updated it `account/contacts`